### PR TITLE
feature: driver/yokogawa gs200

### PR DESCRIPTION
**Pull request by nataliejpg**
Changes proposed in this pull request:
- adds very basic functionality for basic yokogawa voltage source 

@WilliamHPNielsen

### CODE REVIEW

File: qcodes/instrument_drivers/yokogawa/GS200.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    This is the qcodes driver for the Yokogawa GS200 voltage and current source
+
+
+    TODO:(nataliejpg)
+    - add current functionality (mode settings)
+    """
+
+    def __init__(self, name, address, reset=False, **kwargs):
+        """
+        Args:
```
Comment (giulioungaretti) [inline qcodes/instrument_drivers/yokogawa/GS200.py]: 🎠   Class docstring is split, and should all be in under the class and not on the init function.
Reset  attr needs to be described. Also, is it used ? 

See http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for an example.

Comment (nataliejpg) [inline qcodes/instrument_drivers/yokogawa/GS200.py]: docstring moved. I think it's nice to have a reset function, it resets the instrument (not sure it's worth detailing the default settings...). Thought? @giulioungaretti

**Comment by jenshnielsen** [comment]
Would it be ok to merge these by squash? It would be nice to not have the ATS and revert commits going in

**Comment by giulioungaretti** [comment]
@jenshnielsen I always squash and merge, it's the default for qcodes.

**Comment by jenshnielsen** [comment]
Ok missed that somehow.

**Pull request merged**